author/task.py

```diff
@@ -524,7 +529,8 @@ def _add_command(self, task):
                 service=",".join(task.services),
                 tags=task.tags,
                 retryrc=task.retryrc,
-                local=local
+                local=local,
+                envkey=self._get_serialized_envkey() if has_env else []
```

Comment:
Why does this expect a list? Could we pass multiple env keys to the task/job object?

Reply:
I don't know why it expects it, but `envkey` is always a `StringListAttribute`.

![image](https://user-images.githubusercontent.com/25324713/144283389-3585acee-fd7a-4e56-919f-8141c28d10d2.png)